### DISCUSSION TITLE
Flags discussion

### DISCUSSION THREAD

**Discussion by tlizri** [AlexProgrammerDE/balena-minecraft-server#188] [General]
Hi all,

Since the last update of the project, I was investigating about ZCG and Aikar flags and understand how it works.
There is no information on this github and i was a little confuse about how to use it. Now, I know that with the device variables `ZGC_FLAGS` and `AIKAR_FLAGS` we can activate this function in the startup of the server, so it's time to trying them but, which is better for the server? Paper in 1.18.1 suffer from loading chunks lag, specially with boat travels.

I would like more users to write them experience with the flags.

**Comment by viedok** [comment]
Hi!
I have personally tested both but as good as aikar's flags are supposed to be, raspberry pi's just aren't typical machines for minecraft servers. I observed that with aikar's, the server ram would fill up all the way, quite worrying if you ask me. Since I have never looked more into this one, I am not 100% sure that this is bad. I thought that maybe it way just allocating it making the balena dashboard think it's all used ?...
Anyways it scared me a bit so I've been using ZGC for the past year on my rpi (before its official implementation I had forked the project to add this feature) and it works great ! I rarely have more than 1g of ram usage and my server seems to handle it pretty good for a playercount that varies between 1-6.
Hope this helps :)

if you need more help you can contact me on discord : taedium#6515
